src/specs/VideoNativeComponent.ts

```diff
@@ -373,45 +369,45 @@ export interface VideoNativeProps extends ViewProps {
   onVideoTracks?: DirectEventHandler<OnVideoTracksData>; // android
 }
 
-export type VideoComponentType = HostComponent<VideoNativeProps>;
-
-export type VideoSaveData = {
-  uri: string;
-};
-
-export interface VideoManagerType {
-  save: (option: object, reactTag: number) => Promise<VideoSaveData>;
-  seek: (option: Seek, reactTag: number) => Promise<void>;
-  setPlayerPauseState: (paused: boolean, reactTag: number) => Promise<void>;
+export interface NativeVideoCommands {
+  seek: (
+    viewRef: React.ElementRef<HostComponent<VideoNativeProps>>,
+    time: Float,
+    tolerance?: Float,
+  ) => void;
+  setPlayerPauseState: (
+    viewRef: React.ElementRef<HostComponent<VideoNativeProps>>,
+    paused: boolean,
+  ) => void;
+  setVolume: (
+    viewRef: React.ElementRef<HostComponent<VideoNativeProps>>,
+    volume: Float,
+  ) => void;
+  setFullScreen: (
+    viewRef: React.ElementRef<HostComponent<VideoNativeProps>>,
+    fullScreen: boolean,
+  ) => void;
   setLicenseResult: (
+    viewRef: React.ElementRef<HostComponent<VideoNativeProps>>,
     result: string,
     licenseUrl: string,
-    reactTag: number,
-  ) => Promise<void>;
+  ) => void;
   setLicenseResultError: (
+    viewRef: React.ElementRef<HostComponent<VideoNativeProps>>,
     error: string,
     licenseUrl: string,
-    reactTag: number,
-  ) => Promise<void>;
-  setVolume: (volume: number, reactTag: number) => Promise<void>;
-  getCurrentPosition: (reactTag: number) => Promise<number>;
-  setFullScreen: (fullScreen: boolean, reactTag: number) => Promise<void>;
-}
```

Comment:
nit: 
let's move `React.ElementRef<HostComponent<VideoNativeProps>>` into two types
1. `NativeVideoComponentType` for `HostComponent<VideoNativeProps>`
2. `NativeVideoComponentRefType` for `React.ElementRef<NativeVideoComponentType>`
(or you can name it whatever you want)

Just to not repeat it